src/jscpd.coffee

```diff
@@ -32,26 +33,21 @@ class jscpd
       debug: off
       files: null
       exclude: null
-    , options
+    , config
 
-    optionsNew = _.extend optionsNew, config
     for key, value of options
-      if value is not null then optionsNew[key] = value
+      if not (value is null)
+        optionsNew[key] = value
 
     if typeof optionsNew.languages is 'string' then optionsNew.languages = optionsNew.languages.split ','
 
-    if config.path
-      optionsNew.path = path.normalize "#{cwd}/#{config.path}"
-      cwd = options.path
-
     optionsNew.extensions = TokenizerFactory::getExtensionsByLanguages(optionsNew.languages)
-
     return optionsNew
 
-  run: (options)->
-    cwd = options.path
-    config = @readConfig("#{cwd}/.cpd.yaml") || @readConfig("#{cwd}/.cpd.yml") || {}
+  run: (options) ->
+    config = @readConfig(".cpd.yaml") || @readConfig(".cpd.yml") || {}
     options = @prepareOptions options, config
+    options.path = options.path or process.cwd();
```

Comment:
If path was not provided by cmd line options or the config file, fallback to process.cwd here.